test/e2e-common.sh

```diff
@@ -195,6 +210,19 @@ function install_head() {
   kubectl replace -f ./test/config/100-config-kafka-features.yaml
 }
 
+function install_control_plane_from_source() {
```

Comment:
As far as I can tell, you aren't calling this function anywhere? Would you be able to call this function as well so that the control plane components which are being build are then applied into the cluster?